_posts/2018-03-13-how-nx-affected-apps-works.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Understanding how Nx identifies updated apps
+published: true
+author: kwintenp
+description: A deep dive into the implementation of how Nx knows which apps are affected by a PR
+layout: post
+navigation: True
+date:   2018-03-13
+tags: Angular Nx
+subclass: 'post'
+categories: 'kwintenp'
+logo: 'assets/images/strongbrewlogo.png'
+published: true
+disqus: true
+cover: 'assets/images/cover/cover10.jpg'
+---
+
+Nx from Nrwl is a collection of tools that can help us build Angular applications using a monorepo and all the benefits that brings. In essence, Nx is a set of schematics that work on top of the @angular/cli. These schematics can be used to create apps and libs inside of a single @angular/cli project. Having multiple apps is supported by default and Nx leverages this feature and makes the process a little easier.
```

Comment:
and all the benifits that brings sounds weird

Reply:
refer to schematics blog or documentation

Reply:
Fixed and fixed.